.github/workflows/7_publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish 
+
+on:
+  workflow_dispatch:
+    inputs:
+      asset_name:
+        description: 'Name of the .vsix file in the release'
+        required: true
+      tag_name: 
+        description: 'The release version to fetch from.'
+        required: true
+      
+
+env:
+  AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN: ${{ secrets.AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN }}
+
+jobs:
+  marketplace:
+    name: Publish to marketplace
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Install Dependencies
+        run: npm install
+      - uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: "prisma/language-tools"
+          version: "tags/${{ github.event.inputs.tag_name }}"
+          file: ${{ github.event.inputs.asset_name }}
+          token: ${{ secrets.GH_TOKEN }}
```

Comment:
We figure a way to run tests for VSIX right? Does it make sense to test VSIX again before uploading it to marketplaces? To ensure that the packaging happened correctly. We have had a couple of related bugs.

Reply:
No, I don't think this is do-able here, we do have e2e tests for those in another repo for that case.

Reply:
Would that e2e test be taken into consideration here automatically?

Reply:
@janpio  What do you mean?

Reply:
Would a failing e2e test for the vsix file stop publishing it in any way?

Reply:
No, since the e2e tests are run in a different repo after the extension is published already. The e2e tests before publishing will stop publishing if those fail.